### ISSUE TITLE
sys.path does not contain CWD when I try calling py file which import from current dir?

### ISSUE THREAD

**Issue by ClericPy** [linkedin/shiv#154]
a.py
```python
import sys
print(sys.path)
from b import c

c()
```

b.py
```python
def c():
    from bottle import __version__
    print('bottle', __version__)
```

run in shell:

> shiv -o app.pyz bottle

> python app.pyz a.py

How could I `import b` besides copy b.py into site-package path with `--site-packages` arg?

**Comment by lorencarvalho** [comment]
Hi @ClericPy,

The first element of your `sys.path` is going to be the script or executable you invoked (in this case, `app.pyz`). This is pretty standard behavior for a `zipapp`:

```
darwin ~ $ mkdir test
darwin ~ $ echo "print('hello world')" >> test/__main__.py
darwin ~ $ python3 -m zipapp test
darwin ~ $ python3 -i ./test.pyz
hello world
>>> import sys
>>> sys.path[0]
'./test.pyz'
```

Besides copying `b` into your site-packages, you could add $CWD to your sys.path prior to importing.

**Comment by ClericPy** [comment]
@lorencarvalho 

Thank you very much for your answer, and I am sorry for the unclear description of the problem. In fact, my original intention was to use shiv to package a pyz instead of a virtual environment (venv), so the script actually executed is not pyz but the later a.py.
And using shiv .pyz file as a portable venv is a good choice but $CWD is actually the temp folder of `.shiv` cache. This is a common sense mistake I made.

You are right, `.shiv` folder has been added to sys.path and a.py is not the executed py-script in this cmd `python3 venv.pyz a.py`.

This is quite a bad issue...
